Shared/Samples/Augment reality to collect data/AugmentRealityToCollectDataView.swift

```diff
@@ -0,0 +1,210 @@
+            .calibrationButtonAlignment(.bottomLeading)
+            .onCalibratingChanged { isPresented in
+                scene.baseSurface.opacity = isPresented ? 0.5 : 0
+            }
+            .onSingleTapGesture { _, scenePoint in
+                graphicsOverlay.removeAllGraphics()
+                canAddFeature = true
+                
+                // Add feature graphic.
+                graphicsOverlay.addGraphic(Graphic(geometry: scenePoint))
+                statusText = "Placed relative to ARKit plane"
+            }
+            .task {
+                do {
+                    try await featureTable.load()
+                } catch {
+                    self.error = error
+                }
+            }
+            .overlay(alignment: .top) {
+                Text(statusText)
+                    .multilineTextAlignment(.center)
+                    .frame(maxWidth: .infinity, alignment: .center)
+                    .padding(8)
+                    .background(.regularMaterial, ignoresSafeAreaEdges: .horizontal)
+            }
+            Divider()
+        }
+        .toolbar {
+            ToolbarItemGroup(placement: .bottomBar) {
+                Button {
+                    treeHealthSheetIsPresented = true
+                } label: {
+                    Image(systemName: "plus")
+                        .imageScale(.large)
+                }
+                .disabled(!canAddFeature)
+                .actionSheet(isPresented: $treeHealthSheetIsPresented) {
+                    ActionSheet(
+                        title: Text("Add tree"),
+                        message: Text("How healthy is this tree?"),
+                        buttons: [
+                            .default(Text("Healthy")) {
+                                addFeature(treeHealth: .healthy)
+                            },
+                            .default(Text("Distressed")) {
+                                addFeature(treeHealth: .distressed)
+                            },
+                            .default(Text("Dead")) {
+                                addFeature(treeHealth: .dead)
+                            },
+                            .cancel()
+                        ])
+                }
+            }
+        }
+        .errorAlert(presentingError: $error)
+    }
+    
+    /// Adds a feature to represent a tree to the tree survey service feature table.
+    /// - Parameter treeHealth: The health of the tree.
+    private func addFeature(treeHealth: TreeHealth) {
+        guard let featureGraphic = graphicsOverlay.graphics.first,
+              let featurePoint = featureGraphic.geometry as? Point else { return }
+        
+        statusText = "Adding feature"
+        
+        // Create attributes for the new feature.
+        let featureAttributes = [
+            "Health": treeHealth.rawValue,
+            "Height": 3.2,
+            "Diameter": 1.2
+        ] as [String: Any]
+        
+        Task {
```

Comment:
`makeFeature` isn't asynchronous, so should the task be created inside the `if` statement?